presto-tpch/src/test/java/com/facebook/presto/tpch/statistics/TableStatisticsRecorder.java

```diff
@@ -72,8 +72,8 @@
                 return column.getInteger(row);
             case DATE:
                 return column.getDate(row);
-            case DOUBLE:
-                return column.getDouble(row);
+            case DECIMAL:
+                return column.getIdentifier(row);
```

Comment:
did you mean `long`?

Reply:
``getIdentifier`` returns the long value of the column, should it not work for this case?